<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="toggle_check_exit">
            <property name="text">
             <string>Confirm exit while emulation is running</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_new_mode">
            <property name="text">
             <string>Enable New Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_console">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If this is disabled and the software keyboard implementation is StdIn, Citra will freeze when a title starts software keyboard.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Show Console</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_sd_card_root">
            <item>
             <widget class="QLabel" name="label_sd_card_root">
              <property name="text">
               <string>SD card root</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sd_card_root">
              <property name="toolTip">
               <string>Empty for user directory/sdmc</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_sd_card_root_empty">
              <property name="text">
               <string>Empty</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="button_sd_card_root">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_interface_language">
            <item>
             <widget class="QLabel" name="language_label">
              <property name="text">
               <string>Interface language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="language_combobox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_swkbd_implementation">
            <item>
             <widget class="QLabel" name="label_swkbd_implementation">
              <property name="text">
               <string>Software keyboard implementation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combobox_swkbd_implementation">
              <item>
               <property name="text">
                <string>StdIn</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Performance</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="toggle_cpu_jit">
            <property name="text">
             <string>Enable CPU JIT</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Region:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="region_combobox">
              <item>
               <property name="text">
                <string>Auto-select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">JPN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">USA</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">EUR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">AUS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">CHN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">KOR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">TWN</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="theme_group_box">
       <property name="title">
        <string>Theme</string>
       </property>
       <layout class="QHBoxLayout" name="theme_qhbox_layout">
        <item>
         <layout class="QVBoxLayout" name="theme_qvbox_layout">
          <item>
           <layout class="QHBoxLayout" name="theme_qhbox_layout_2">
            <item>
             <widget class="QLabel" name="theme_label">
              <property name="text">
               <string>Theme:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="theme_combobox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Logging</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_log_filter">
            <property name="text">
             <string>Global Log Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="log_filter_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="open_log_button">
            <property name="text">
             <string>Open Log Location</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Hotkeys</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="GHotkeysDialog" name="widget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GHotkeysDialog</class>
   <extends>QWidget</extends>
   <header>hotkeys.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
